templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete page</h1>
+  <hr>
+  <strong>ARE YOU SURE?</strong>
+  <p><i>Deleting a car will delete all drivers with that car!</i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input class="btn btn-danger" type="submit" value="DELETE">

```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on 
-- checklist